src/value.rs

```diff
@@ -96,20 +96,20 @@ impl Value {
     #[cfg(not(feature = "enable_unstable_features_that_may_break_with_minor_version_bumps"))]
     pub(crate) fn from_events<T>(events: T) -> Result<Value, Error>
     where
-        T: IntoIterator<Item = Result<Event, Error>>,
+        T: IntoIterator<Item = Result<Event<'static>, Error>>,
     {
         Builder::new(events.into_iter()).build()
     }
 
     /// Converts a `Value` into an `Event` iterator.
     #[cfg(feature = "enable_unstable_features_that_may_break_with_minor_version_bumps")]
-    pub fn into_events(self) -> IntoEvents {
+    pub fn into_events(&self) -> IntoEvents {
         IntoEvents::new(self)
     }
 
     /// Converts a `Value` into an `Event` iterator.
     #[cfg(not(feature = "enable_unstable_features_that_may_break_with_minor_version_bumps"))]
-    pub(crate) fn into_events(self) -> IntoEvents {
+    pub(crate) fn into_events(&self) -> IntoEvents {
```

Comment:
ultimately we would rename this to `to_events` or something similar, but I was trying to keep the changes minimal.